README.md

```diff
@@ -64,12 +64,20 @@ Ctws.user_class = "Account"
 The application `User` model **must have the `email` attribute**.
 
 The **`password` is optional** by default a user is validated with password, for `password` validation [`ActiveModel::SecurePassword::InstanceMethodsOnActivation authenticate`](https://apidock.com/rails/v4.2.7/ActiveModel/SecurePassword/InstanceMethodsOnActivation/authenticate) and [`Devise::Models::DatabaseAuthenticatable#valid_password?`](http://www.rubydoc.info/github/plataformatec/devise/Devise%2FModels%2FDatabaseAuthenticatable:valid_password%3F) User instance methods are supported.
+<!--
+To opt out the user validation with the password change it by creating or editing the `ctws.rb` initializer file in `config/initializers` and put this content in it: -->
 
-To opt out the user validation with the password change it by creating or editing the `ctws.rb` initializer file in `config/initializers` and put this content in it:
+To enable user validation with a password and verify that the user meets certain characteristics (such as being confirmed), you must create or edit the `ctws.rb` initializer file  at `config/initializers` and write something similar to the following:
 
 ```ruby
```

Comment:
I would specify that it is Devise specific, and would add also the Authlogic example please.